amy/consents/migrations/0007_term_short_description.py

```diff
@@ -0,0 +1,45 @@
+# Written by @elichad 2023-03-15
+
+from django.db import migrations, models
+
+term_descriptions = {
+    "may-contact": "May contact",
+    "public-profile": "Consent to making profile public",
+    "may-publish-name": "Consent to include name when publishing lessons",
+    "privacy-policy": "Privacy policy agreement",
+}
+
+
+def set_term_descriptions(apps, schema_editor):
+    Term = apps.get_model("consents", "Term")
+    for term in Term.objects.all():
+        term.short_description = term_descriptions.get(
+            term.slug, term.slug.replace("-", " ").capitalize()
+        )
+        term.save(update_fields=["short_description"])
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("consents", "0006_auto_20210614_1234"),
+    ]
+
+    operations = [
+        # add field and make it temporarily nullable
+        migrations.AddField(
+            model_name="term",
+            name="short_description",
+            field=models.CharField(null=True, max_length=100),
```

Comment:
You should be able to create the field with `""` as default value, therefore there should be no need switch `null=True` and then `null=False` in a migration.

Reply:
Wouldn't you then have to do the same thing with `blank=True` and `blank=False` instead? I don't think we want this field to be permitted to be blank as it may appear in the UI through lines like this https://github.com/carpentries/amy/pull/2361/files#diff-a97c97acc5cd76b1c48999bacdc2b399b62f678806ab48b559dcc0944c366536R278

Reply:
I *think* `blank` is only applied when creating model instances. You definitely should be fine if you provide one-off value for the migration (default value of `""`). `makemigrations` may even ask you for it.